DQM/SiPixelPhase1Common/python/MessageLogger_cfi.py

```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
+
+MessageLogger = cms.Service(
```

Comment:
this overrides the central definition of MessageLogger when included in standard configuration.
This should not be done.